style.css

```diff
@@ -5,17 +5,22 @@ html{
 body{
 	font-family: helvetoca. arial sans-serif;
 	margin: 0;
+    background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
 }
 
 .grid-container{
 	display: grid;
-	grid-template-columns: 1fr 2fr 2fr 2fr 1fr;
+	grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;

```

Comment:
Addition of extra columns allowed for more flexibility for reponsive design as it allowed me to have me to fit more images per screen size instead a of maybe 3 when I wanted 4 [at a specific page width]